style.css

```diff
@@ -657,3 +683,726 @@ ul li {
   background-color: #fff;
   color: black;
 }
+
+@media (min-width: 768px) {
+  body {
+    height: 5125px;
+    width: 1440px;
+    left: 0;
+    top: 0;
+    border-radius: 0;
+    position: relative;
+    background: #dfe1e6;
+  }
+
+  /* .container {
+    height: 916px;
+    width: 1440px;
+    left: 0;
+    top: 1px;
+    position: absolute;
+    background: #0d4ed1;
+    border-radius: 0 0 0 10%/0 0 0 10%;
+  } */
+  .container {
+    position: relative;
+    width: 1440px;
+    height: 5125px;
+    background: #f4f5f7;
+  }
+
+  header {
+    position: absolute;
+    width: 1440px;
+    height: 92px;
+    left: 0;
+    top: 0;
+    padding-top: 40px;
+    border-radius: 0;
+    background-color: #fff;
+  }
+
+  .container header,
+  .main {
+    position: absolute;
+    width: 1440px;
+    height: 916px;
+    left: 0;
+    top: 1px;
+    background: #fff;
+    border-radius: 0 0 0 10%/0 0 0 10%;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .navbar-links {
+    position: absolute;
+    display: block;
+    padding: 0;
+    gap: 8px;
+    width: 258px;
+    height: 40px;
+    left: 1039px;
+    top: 16px;
+  }
+
+  .mylogo {
+    width: 76px;
+    height: 20px;
+    margin-left: 155px;
+    margin-top: -10px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .navbar-links ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 18px;
+    position: absolute;
+    width: 258px;
+    height: 40px;
+    left: 0;
+    top: 0;
+  }
+
+  .main {
+    background-image: url(img/Header\ bg@2x.png);
+    background-size: 100% 100%;
+    background-position: 100% 100%;
+    height: 721.4596557617188px;
+    width: 1397.3985595703125px;
+    left: 34.6015625px;
+    top: 119px;
+    border-radius: 0;
+  }
+
+  main h1 {
+    width: 920px;
+    height: 122px;
+    margin-left: 220px;
+    margin-top: 120px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+  }
+
+  .mytxt {
+    width: 920px;
+    height: 98px;
+    margin-left: 220px;
+    margin-top: 360px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .connect {
+    margin-left: 220px;
+  }
+
+  .social {
+    margin-left: 220px;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 142px;
+    gap: 142px;
+    position: relative;
+    width: 1440px;
+    height: 2694px;
+    left: 0;
+    top: 917px;
+  }
+
+  .card1 {
+    position: absolute;
+    height: 496px;
+    width: 1156px;
+    left: 142px;
+    top: 142px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+  }
```

Comment:
- [ ] **[OPTIONAL]** Kindly, make sure the project that your project elements do not overflow outside of the page. You can do that by avoiding as much as possible the use of `position: absolute (prefer layout element with `flex` or `grid` only),  `px` for `height` and `width` (prefer use `%` or `em`).  

**Your project**
![image](https://user-images.githubusercontent.com/74411135/236565152-dddc563e-78f8-4275-9e8d-e569801c8c6f.png)
![image](https://user-images.githubusercontent.com/74411135/236565204-f0b4b805-eeea-4c9e-9331-a7a125111251.png)
![image](https://user-images.githubusercontent.com/74411135/236565286-3795c22d-f240-485e-931c-bbe5ce3adcf3.png)

This will highly improve the user experience 📱 😄